pkg/controller/flightrecorder/flightrecorder_controller.go

```diff
@@ -171,6 +171,21 @@ func (r *ReconcileFlightRecorder) Reconcile(request reconcile.Request) (reconcil
 		return reconcile.Result{}, err
 	}
 
+	// Retrieve list of available templates
+	log.Info("Listing templates for pod", "name", targetPod.Name, "namespace", targetPod.Namespace)
+	templates, err := cjfr.ListTemplates(targetAddr)
+	if err != nil {
+		log.Error(err, "failed to list templates")
+		return reconcile.Result{}, err
+	}
+
+	// Update Status with templates
+	instance.Status.Templates = templates
+	err = r.Client.Status().Update(ctx, instance)
```

Comment:
Is there an issue with calling Status().Update twice (done above for event types)? It could be joined with the code above and then a single Update could be called, im not sure which is better.

Reply:
There shouldn't be an issue other than performance. The client we use reads from a cache, but writes directly to the cluster [1]. So this would mean two writes instead of one. I would remove the first status update, and write both events and templates at once. 

[1] https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.6.1#hdr-Clients_and_Caches